packages/block-editor/src/components/block-list-appender/style.scss

```diff
@@ -1,22 +1,22 @@
 // These styles are only applied to the appender when it appears inside of a block.
 // Otherwise the default appender may be improperly positioned in some themes.
 .block-editor-block-list__block .block-list-appender {
-	margin: $block-padding 0;
+	margin: $grid-unit-10 0;
 
 	// Add additional margin to the appender when inside a group with a background color.
 	// If changing this, be sure to sync up with group/editor.scss line 13.
 	.has-background & {
-		margin: ($block-padding*2 + $block-spacing) $block-padding;
+		margin: ($grid-unit-20 + $block-spacing) $grid-unit-10;
 	}
 }
 
 .block-list-appender.is-drop-target > div::before {
 	content: "";
 	position: absolute;
-	right: -$block-padding;
-	left: -$block-padding;
-	top: -$block-padding;
-	bottom: -$block-padding;
```

Comment:
I wonder if this variable is still needed at all?

Reply:
<img width="776" alt="Screenshot 2020-03-04 at 14 20 01" src="https://user-images.githubusercontent.com/1204802/75883429-41430900-5e23-11ea-863f-42b2602732f3.png">

It is. 

I recall us recently removing a min-height on the "fake appender", btw. That makes it 22px tall, when it should be 28px tall.